example/android/app/build.gradle

```diff
@@ -204,4 +202,4 @@ task copyDownloadableDepsToLibs(type: Copy) {
 }
 
 apply from: file("../../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle");
-applyNativeModulesAppBuildGradle(project, "../..")
+applyNativeModulesAppBuildGradle(project);
```

Comment:
why this change? does autolinking on android still work?

Reply:
yes, the second argument is deprecated on rn 62 applying the `"@react-native-community/datetimepicker": "./"` in package.json the autolink works successfull